src/log.cpp

```diff
@@ -235,7 +235,7 @@ namespace rhost {
                     msgbox_text += c;
                 }
                 
-                MessageBoxA(HWND_DESKTOP, msgbox_text.c_str(), "Microsoft R Host Process fatal error", MB_OK | MB_ICONERROR);
+                //MessageBoxA(HWND_DESKTOP, msgbox_text.c_str(), "Microsoft R Host Process fatal error", MB_OK | MB_ICONERROR);
```

Comment:
I removed this because we detour all message boxes to VS, and it already receives an error message (and shows it to the user) through other channels. This was originally here for debugging the host standalone.